### PR TITLE
feat: logs support on dashboard drilldown

### PR DESCRIPTION
**Pull request by ktx-vaidehi**
- #5194

## Summary by CodeRabbit

- **New Features**
  - Introduced new logging options in drilldown functionality, allowing users to select between "Auto" and "Custom" logs modes.
  - Added a new query editor for custom log queries.
  - Enhanced drilldown navigation to logs with improved error handling and logging.
  - Added optional fields for logging configurations in the drilldown data structure.
  - Improved responsiveness of the dashboard component with new computed properties and watch functions.

- **Bug Fixes**
  - Improved validation logic for drilldown forms to accommodate new logging options.

### CODE REVIEW

**Review by ktx-kirtan — APPROVED** [review]

File: web/src/components/dashboards/PanelSchemaRenderer.vue [hunk 1]
```diff
@@ -680,9 +675,37 @@ export default defineComponent({
...
+    onBeforeMount(async () => {
+      await importSqlParser();
+    });
+
+    const importSqlParser = async () => {
+      const useSqlParser: any = await import("@/composables/useParser");
+      const { sqlParser }: any = useSqlParser.default();
+      parser = await sqlParser();
+    };
+
```
[the remark this answers is not in the document]
Comment (ktx-kirtan) [reply web/src/components/dashboards/PanelSchemaRenderer.vue:688]: We do not need the loading/error state for the parser

File: web/src/components/dashboards/PanelSchemaRenderer.vue [hunk 2]
```diff
@@ -679,10 +675,128 @@ export default defineComponent({
...
```
[the remark this answers is not in the document]
Comment (ktx-vaidehi) [reply web/src/components/dashboards/PanelSchemaRenderer.vue:691]: We don’t need this

File: web/src/components/dashboards/PanelSchemaRenderer.vue [hunk 3]
```diff
@@ -679,10 +675,128 @@
...
+      if (breakdownColumn && breakdownValue) {
+        const breakdownCondition = `${breakdownColumn} = '${breakdownValue}'`;
+        whereClause += whereClause
+          ? ` AND ${breakdownCondition}`
+          : ` WHERE ${breakdownCondition}`;
+      }
+
+      return whereClause;
+    };
```
[the remark this answers is not in the document]
Comment (ktx-vaidehi) [reply web/src/components/dashboards/PanelSchemaRenderer.vue:765]: This can cause issue

**Comment by charl3y15** [comment]
How do I use this? I don't see any documentation

**Comment by ktx-vaidehi** [comment]
> How do I use this? I don't see any documentation

Added documentation #5194

**Comment by charl3y15** [comment]
> > How do I use this? I don't see any documentation
> 
> Added documentation #5194

I only see this
{8B2E8CD1-1744-435F-89DF-B3ED5298E8AA}

**Pull request merged**